tangelo/algorithms/variational/tests/test_sa_vqe_solver.py

```diff
@@ -137,6 +138,31 @@ def test_simulate_h2(self):
         energy = vqe_solver_2.simulate()
         self.assertAlmostEqual(exact_energies[2], energy, delta=1.e-3)
 
+    def test_projected_simulate_h2(self):
+        """Run SA-VQE on H2 molecule, with UpCCGSD ansatz, JW qubit mapping, ref_states and exact simulator.
+        Followed by deflation of two states calculated to determine next excited state.
```

Comment:
I see no explicit mention of projection here. Is this implied by the second sentence ?
If you think this was clear enough, you can just resolve this comment.